.buildkite/pipeline.yaml

```diff
@@ -26,7 +26,6 @@ steps:
 
   - label: ":mac: test-static-sanitized.sh (master only)"
     command: .buildkite/test-static-sanitized.sh
-    branches: "master"
```

Comment:
Don't forget to remove this before merging.

Reply:
I'm planning to get a re-review from trevor before merging. Am removing this and commenting one of the bazel changes.